docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -2195,10 +2239,71 @@ Segment:
     description: 'This class represents a segment.'
     type: BaseObject
     properties:
-        id: integer
-        identifier: string
-        name: string
-        SegmentGroup: SegmentGroup
+        id:
+            description: 'A unique numeric identifier for the segment.'
+            type: integer
+        identifier:
+            description: 'A unique numeric identifier for the segment.'
+            type: string
+        name:
+            description: 'A name of the segment.'
+            type: string
+        SegmentGroup:
+            description: 'A reference to the segment group to which this segment belongs.'
+            type: object
+            properties:
+                media-type:
```

Comment:
Because the `media-type` and href are attributes for XML, but properties for JSON, please use the following syntax:
https://github.com/ibexa/documentation-developer/blob/6afb6168e06d89e9090e721dc5f3c1ace86b7698/docs/api/rest_api/rest_api_reference/input/ibexa-types.raml#L72-L82

Reply:
Right, I haven't noticed that in other examples.